hearth/lib/hearth/http/error_parser.rb

```diff
@@ -30,22 +30,17 @@ class ErrorParser
       #
       # @param [Array<Class<ApiError>>] errors Array of Error classes
       #   modeled for the operation.
-      #
-      # @param [callable] error_code_fn Protocol specific function
-      #   that will return the error code from a response, or nil if
-      #   there is none.
-      def initialize(error_module:, success_status:, errors:, error_code_fn:)
+      def initialize(error_module:, success_status:, errors:)
```

Comment:
Nit: add to documenation of error_module, it must contain definition of error code function